packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -46,11 +56,14 @@ const Conversation = WebexPlugin.extend({
    * @returns {Promise<Activity>}
    */
   add(conversation, participant, activity) {
-    return this._inferConversationUrl(conversation)
+    const url = this.getConvoUrl(conversation);
+    const convoWithUrl = Object.assign({}, conversation, {url});
+
+    return Promise.resolve()
```

Comment:
Since `this.webex.internal.user.asUUID` returns a promise, you could omit this line and just return:
```js
return this.webex.internal.user.asUUID
```